index.html

```diff
@@ -420,14 +419,22 @@
           settings</var>' <a>top-level browsing context</a>'s <a>origin</a>
         </li>
       </ol>
+      <dl class='idl' title='dictionary PermissionStorage'>
+        <dt>
+          required PermissionState state
+        </dt>
+      </dl>
+      <p class="note">
+        <a>PermissionStorage</a> instances are never exposed to Javascript.
+      </p>
       <p>
         The steps to <dfn>retrieve a permission storage entry</dfn> of a
```

Comment:
This is the sort of cleanup that I want to do, but in a separate patch, so this series doesn't grow any more. We currently have [retrieve a permission storage entry](https://rawgit.com/jyasskin/permissions/allow-choosers/index.html#dfn-retrieve-a-permission-storage-entry) that's only called from [retrieve the permission storage](https://rawgit.com/jyasskin/permissions/allow-choosers/index.html#dfn-retrieve-the-permission-storage), so maybe I can eliminate an unnecessary layer entirely, instead of just renaming it.